src/pages/blocks/Blocks.tsx

```diff
@@ -82,38 +88,27 @@ const Blocks: React.FC<MatchParams> = props => {
   const blockState = useSelector(({ block }: { block: BlockState }) => block)
   const width = useWindowWidth()
 
-  function loadMore(): void {
-    const nextPage = blockState.page + 1
-    dispatch(fetchBlocks(nextPage))
-  }
   const history = useHistory()
   const { chain, network: networkParam } = useParams<MatchParams>()
   const { protocol, handleSetFilterData, network } = useFilterStateWithHistory(
     history,
     chain,
     networkParam,
   )
+  const [perPage, setPerPage] = useState<number>(60)
```

Comment:
This default value is not always true, if you change the filter to something other than all and reload the page, the filter will start if the last change but the state not. So I recommend you start the create a useEffect. 
```suggestion
  const [perPage, setPerPage] = useState<number>()
  
  useEffect(() => {
     if (network !== "all") {
       setPage(15)
       return
     }
     
     setPage(60)
  }, [network])
```